deploy/crds/keycloak.org_keycloakbackups_crd.yaml

```diff
@@ -69,6 +69,13 @@ spec:
                 flag to true. Potentially, it will be possible to restore a single
                 backup multiple times."
               type: boolean
+            container:
```

Comment:
The Operator uses 4 images at the moment:
- a database image - Postgresql
- an Integreately Backup container
- a Keycloak image
- an Init image

The proposed implementation seems to support only Keycloak, which is not enough. We should allow to override all the images.

The `Container` struct doesn't look like a good fit as it is defined as follows:

```go
type Container struct {
   Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
   Image string `json:"image,omitempty" protobuf:"bytes,2,opt,name=image"`
}
```

So there's no obvious way to override any of the 4 images. 

In the [Keycloak Dev discussion](https://groups.google.com/forum/#!topic/keycloak-dev/OvgA_CNfgok) (linked from [KEYCLOAK-12409](https://issues.redhat.com/browse/KEYCLOAK-12409)) I proposed something like this:

```yaml
apiVersion: keycloak.org/v1alpha1
kind: Keycloak
metadata:
  name: example-keycloak
  labels:
    app: sso
spec:
  instances: 1
  # Naming needs to be consistent with realmOverrides from KeycloakRealmCR
  # All fields are optional
  imageOverrides:
    operator: <custom image>
    postgresql <custom image>
    backup: <custom image>
``` 

I think it may work better as all images have a corresponding field (however, the example above misses init container and Keycloak images). Also, `imageOverrides` is similar to `realmOverrides` we already used in the code.

What do you think about this idea? Would you like to implement this changes?